mlflow/pipelines/utils/step.py

```diff
@@ -111,16 +112,8 @@ def get_pandas_data_profile(data_frame, title: str):
     :param title: String, the title of the data profile.
     :return: a data profiling object such as Pandas profiling ProfileReport.
     """
-    from pandas_profiling import ProfileReport
-
     if len(data_frame) == 0:
-        return ProfileReport(
-            data_frame,
-            title=title,
-            minimal=True,
-            progress_bar=False,
-            pool_size=_get_pool_size(),
-        )
+        pandas_renderer.get_html([[title, data_frame]])
```

Comment:
Just to confirm that empty dataframe still works, correct?

Reply:
It returns an empty HTML. The tab doesn't break but it won't render anything